mempool/src/error/ban_score.rs

```diff
@@ -254,16 +251,16 @@ impl MempoolBanScore for pos_accounting::Error {
         use pos_accounting::Error as E;
         match self {
             // These may be caused by an out of sync peer
-            // TODO: Can we ban the peer in these cases anyway? Some will be only triggered
-            // after an epoch has elapsed which should be enough time for peer to be up-to-date
-            E::InvariantErrorPoolBalanceAlreadyExists => 0,
-            E::InvariantErrorPoolDataAlreadyExists => 0,
-            E::InvariantErrorDelegationCreationFailedIdAlreadyExists => 0,
             E::AttemptedDecommissionNonexistingPoolBalance => 0,
             E::AttemptedDecommissionNonexistingPoolData => 0,
             E::DelegationCreationFailedPoolDoesNotExist => 0,
             E::DelegateToNonexistingId => 0,
             E::DelegateToNonexistingPool => 0,
```

Comment:
@azarovh Could you check these are all indeed dependent on the current state? Or could we use any of these to ban the peer?

Reply:
I believe these are correct. 

One thing is that `E::AccountingError(_) => 100` shouldn't be generalized. I think there are some errors inside that depend on a state like `DataCreatedMultipleTimes` or `ModifyNonexistingData` and so on. They are most likely overlapped by higher level `pos_accounting` errors but still.

Reply:
> One thing is that `E::AccountingError(_) => 100` shouldn't be generalized. I think there are some errors inside that depend on a state like `DataCreatedMultipleTimes` or `ModifyNonexistingData` and so on. They are most likely overlapped by higher level `pos_accounting` errors but still.

Added `BanScore` for `accounting::Error` (see commit 6334000c50fa7407a08ac3c5163d381225e8095f). Can you take a look whether it seems sensible?